crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -1102,7 +1095,7 @@ TEST_P(RSA_ServiceIndicatorTest, RSASigVer) {
 
   int approved = AWSLC_NOT_APPROVED;
 
-  bssl::UniquePtr<RSA> rsa(RSA_new());
+  RSA *rsa = RSA_new();
```

Comment:
I guess this pointer needs to be freed

Reply:
See response to https://github.com/awslabs/aws-lc/pull/342/files#r768114289
so I think I cannot free it.